### ISSUE TITLE
caching requests to currency api - memoizing

### ISSUE THREAD

**Issue by daymos** [smooth-operators-fac/currency-map#36]

**Comment by g-sam** [comment]
yes! consider branching off `server` to reduce MMM (massive merge mayhem).
I got bogged down making a script to pull down large amounts of historical data. nearly there with it now though!

**Comment by g-sam** [comment]
actually, it's now working! going to make a PR
